services/payments/authorize.go

```diff
@@ -5,27 +5,33 @@ import (
 	"crypto/ed25519"
 	"encoding/hex"
 	"fmt"
+	"os"
 	"strings"
 
 	"github.com/brave-intl/bat-go/libs/httpsignature"
 	paymentLib "github.com/brave-intl/bat-go/libs/payments"
 	"golang.org/x/crypto/ssh"
 )
 
-var validAuthorizerKeys = []string{
-	// @evq
-	"ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIA91/jZI+hcisdAURdqgdAKyetA4b2mVJIypfEtTyXW+ evq+settlements@brave.com",
-	// @sneagan
-	"ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIDfcr9jUEu9D9lSpUnPwT1cCggCe48kZw1bJt+CXYSnh jegan+settlements@brave.com",
-	// @kdenhartog for dev environment only
-	"ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIEY/3VGKsrH5dp3mK5PJIHVkUMWpsmUhZkrLuZTf7Sqr kdenhartog+settlement+dev@brave.com",
+var validAuthorizerKeys = map[string][]string{
+	"production": {},
+	"staging": {
+		// @evq
+		"ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIA91/jZI+hcisdAURdqgdAKyetA4b2mVJIypfEtTyXW+ evq+settlements@brave.com",
+		// @sneagan
+		"ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIDfcr9jUEu9D9lSpUnPwT1cCggCe48kZw1bJt+CXYSnh jegan+settlements@brave.com",
+	},
+	"develop": {
```

Comment:
Maybe `development` would be better? Also, consistent with some other places (one example is SKUs).

Reply:
Good catch. Updated in both PRs.